cloud/docker/docker.py

```diff
@@ -223,6 +223,14 @@
     description:
       - Pass a dict of environment variables to the container.
     default: null
+  env_file:
+    description:
+      - Pass in a path to a file with environment variable (FOO=BAR).
+        If a key value is present in both explicitly presented (i.e. as 'env')
+        and in the environment file, the explicit value will override.
+        Requires docker-py >= 1.4.0.
+    default: null
+    required: false
```

Comment:
:point_up: While we're at it, this doc section should have a `version_added:` key in it.